lib/c-api/src/wasm_c_api/wasi/mod.rs

```diff
@@ -21,7 +21,7 @@ use wasmer_api::{AsStoreMut, Imports, Module};
 use wasmer_wasix::{
     Pipe, PluggableRuntime, WasiEnv, WasiEnvBuilder, WasiFunctionEnv, WasiVersion,
     default_fs_backing, get_wasi_version,
-    runtime::task_manager::{InlineWaker, tokio::TokioTaskManager},
+    runtime::task_manager::{block_on, tokio::TokioTaskManager},
```

Comment:
Use the `block_on` import from `virtual_mio` instead (where is defined)

Reply:
This crate doesn't have a virtual_mio dep